src/Parser/Hunter/Marksmanship/Modules/Talents/TrueAim.js

```diff
@@ -58,19 +66,33 @@ class TrueAim extends Analyzer {
     this._currentStacks = 0;
   }
 
+  on_byPlayer_damage(event) {
+    const spellId = event.ability.guid;
+    if (spellId !== SPELLS.AIMED_SHOT.id && spellId !== SPELLS.ARCANE_SHOT.id) {
+      return;
+    }
+    if (spellId === SPELLS.AIMED_SHOT.id) {
+      this.aimedBonusDmg += getDamageBonus(event, (TRUE_AIM_MODIFIER*this._currentStacks));
+    }
+    if (spellId === SPELLS.ARCANE_SHOT.id) {
+      this.arcaneBonusDmg += getDamageBonus(event, (TRUE_AIM_MODIFIER*this._currentStacks));
+    }
+    this.bonusDmg += getDamageBonus(event, (TRUE_AIM_MODIFIER * this._currentStacks));
+  }
+
   statistic() {
     const percentTimeAtMaxTAStacks = formatPercentage(this.timeAtMaxStacks / this.owner.fightDuration);
-    //only shows if timesDropped is greater than 0, to only show it on encounters where it might matter - we expect full uptime on any ST boss.
-      return (
-        <StatisticBox
-          icon={<SpellIcon id={SPELLS.TRUE_AIM_DEBUFF.id} />}
-          value={`${percentTimeAtMaxTAStacks} %`}
-          label="10 stack uptime"
-          tooltip={`You reset True Aim when you had 3 or more stacks (to exclude trickshot cleaving resets): ${this.timesDropped} times over the course of the encounter. <br />Your total amount of resets (including with trickshot cleaving) was: ${this.totalTimesDropped}.`}
-        />
-      );
-  }
+    return (
+      <StatisticBox
+        icon={<SpellIcon id={SPELLS.TRUE_AIM_DEBUFF.id} />}
+        value={`${percentTimeAtMaxTAStacks} %`}
+        label="10 stack uptime"
+        tooltip={`You reset True Aim when you had 3 or more stacks (to exclude trickshot cleaving resets): ${this.timesDropped} times over the course of the encounter. <br />Your total amount of resets (including with trickshot cleaving) was: ${this.totalTimesDropped}. <br /> True Aim contributed with           ${formatNumber(this.bonusDmg)} - ${this.owner.formatItemDamageDone(this.bonusDmg)}. <br/> Aimed Shot did ${formatPercentage(
```

Comment:
You could re-write the below as:

`<br /> True Aim contributed ${formatNumber(this.bonusDmg)} - ${this.owner.formatItemDamageDone(this.bonusDmg)}. <br/> Aimed Shot contribution: ${formatPercentage(this.aimedBonusDmg/this.bonusDmg)}% / Arcane Shot contribution: ${formatPercentage(this.arcaneBonusDmg/this.bonusDmg)}%.`

Also, the final lines could be treated as just an unordered list.